### ISSUE TITLE
Free Weekends still being counted as $0 deal

### ISSUE THREAD

**Issue by FolkSong** [IsThereAnyDeal/IsThereAnyDeal-Issues#2969]
This is misleading, and messes up the historical low.

Example:

https://isthereanydeal.com/game/deeprockgalactic/history/

**Comment by dooleyj** [comment]
Agreed, it is also annoying, I don't care about free weekends, I just want to know lowest price.  It is a waste of the users time to make them go to the sale site to find out it is just a free weekend.
